### PR TITLE
SAI release notes for 1.12.0 version

### PR DESCRIPTION
**Pull request by kannankvs**
Created draft version of SAI release notes for 1.12.0

### CODE REVIEW

File: doc/SAI_1.12.0_ReleaseNotes.md
Comment (apogedik) [inline doc/SAI_1.12.0_ReleaseNotes.md:1]: Can https://github.com/opencomputeproject/SAI/pull/1816 be added, as well

Comment (kannankvs) [reply doc/SAI_1.12.0_ReleaseNotes.md:1]: @apogedik  The [PR#1816](https://github.com/opencomputeproject/SAI/pull/1816) is still in open state. Is this part of 1.12.0 version of SAI ?

**Comment by apogedik** [comment]
We would like it to be included in 1.12.0.On 20 Jul 2023, at 22:33, kannankvs ***@***.***> wrote:﻿
@kannankvs commented on this pull request.



On doc/SAI_1.12.0_ReleaseNotes.md:
@apogedik  The PR#1816 is still in open state. Is this part of 1.12.0 version of SAI ?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

This electronic communication and the information and any files transmitted with it, or attached to it, are confidential and are intended solely for the use of the individual or entity to whom it is addressed and may contain information that is confidential, legally privileged, protected by privacy laws, or otherwise restricted from disclosure to anyone else. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any use, copying, distributing, dissemination, forwarding, printing, or copying of this e-mail is strictly prohibited. If you received this e-mail in error, please return the e-mail to the sender, delete it from your computer, and destroy any printed copy of it.

**Comment by kannankvs** [comment]
> We would like it to be included in 1.12.0

@apogedik  Okay

**Comment by kannankvs** [comment]
> We would like it to be included in 1.12.0

@apogedik  Can you please merge the PR for the release. Since its in open state, we are unable to add until the PR is merged and until the PR is cherry picked into the release.

**Comment by apogedik** [comment]
I am requesting the right people to review/merge the PR. I am not authorized to merge. On Jul 21, 2023, at 1:35 AM, kannankvs ***@***.***> wrote:﻿

We would like it to be included in 1.12.0

@apogedik  Can you please merge the PR for the release. Since its in open state, we are unable to add until the PR is merged and until the PR is cherry picked into the release.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

This electronic communication and the information and any files transmitted with it, or attached to it, are confidential and are intended solely for the use of the individual or entity to whom it is addressed and may contain information that is confidential, legally privileged, protected by privacy laws, or otherwise restricted from disclosure to anyone else. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any use, copying, distributing, dissemination, forwarding, printing, or copying of this e-mail is strictly prohibited. If you received this e-mail in error, please return the e-mail to the sender, delete it from your computer, and destroy any printed copy of it.

**Comment by kannankvs** [comment]
> I am requesting the right people to review/merge the PR. I am not authorized to merge. On Jul 21, 2023, at 1:35 AM, kannankvs ***@***.***> wrote:﻿ We would like it to be included in 1.12.0 @apogedik Can you please merge the PR for the release. Since its in open state, we are unable to add until the PR is merged and until the PR is cherry picked into the release. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***> This electronic communication and the information and any files transmitted with it, or attached to it, are confidential and are intended solely for the use of the individual or entity to whom it is addressed and may contain information that is confidential, legally privileged, protected by privacy laws, or otherwise restricted from disclosure to anyone else. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any use, copying, distributing, dissemination, forwarding, printing, or copying of this e-mail is strictly prohibited. If you received this e-mail in error, please return the e-mail to the sender, delete it from your computer, and destroy any printed copy of it.



> I am requesting the right people to review/merge the PR. I am not authorized to merge. On Jul 21, 2023, at 1:35 AM, kannankvs ***@***.***> wrote:﻿ We would like it to be included in 1.12.0 @apogedik Can you please merge the PR for the release. Since its in open state, we are unable to add until the PR is merged and until the PR is cherry picked into the release. —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***> This electronic communication and the information and any files transmitted with it, or attached to it, are confidential and are intended solely for the use of the individual or entity to whom it is addressed and may contain information that is confidential, legally privileged, protected by privacy laws, or otherwise restricted from disclosure to anyone else. If you are not the intended recipient or the person responsible for delivering the e-mail to the intended recipient, you are hereby notified that any use, copying, distributing, dissemination, forwarding, printing, or copying of this e-mail is strictly prohibited. If you received this e-mail in error, please return the e-mail to the sender, delete it from your computer, and destroy any printed copy of it.

@rlhui  Can you please help here.

**Review by rlhui — APPROVED** [review]

**Pull request merged**